src/devices/AD5328/README.md

```diff
@@ -0,0 +1,12 @@
+
+# AD5328 - Digital to Analog Convertor

```

Comment:
no need of extra line before the title for the first tile

Reply:
Done